test/test_encrypted_file_mapping.cpp

```diff
@@ -17,7 +17,10 @@
  **************************************************************************/
 
 #include "testsettings.hpp"
-#ifdef TEST_ENCRYPTED_FILE_MAPPING
+
+// FIXME: Unit tests use POSIX not compatible with Visual Studio and 
```

Comment:
This PR won't remove any existing unit tests. It only ignores new incompatible tests. So I don't know. I think I will just rewrite the tests the next few days myself without issue